modules/reports/models/scheduled_reports.php

```diff
@@ -242,7 +242,8 @@ static function update_report_field($id=false, $field=false, $value=false)
 		try {
 			$res = $db->query($sql);
 		} catch (Kohana_Database_Exception $e) {
-			print $e->getMessage();
+			op5log::instance('ninja')->log('error', "Database request failed, $e");
```

Comment:
Isn't `$e->getMessage()` required?

Reply:
You are correct. Fixed in amended commit.